modules/ting_search_context/ting_search_context.install

```diff
@@ -0,0 +1,200 @@
+        'length' => 255,
+      ),
+            // Administrators can also extend the system by creating search
+            // string contexts. Search contexts doesn't have a context and
+            // subject value. Unlike subjects contexts, search string contexts
+            // doesn't have a parent. The context is just the search string.
+      'search' => array(
+        'description' => 'The search string of this context.',
+        'type' => 'varchar',
+        'length' => 255,
+      ),
+    ),
+    'primary key' => array('context_id'),
+        // Enforce unique contexts on the database level.
+    'unique keys' => array(
+      'subject' => array('context', 'subjects'),
+      'search' => array('search'),
+    ),
+  );
+
+  $schema['ting_search_context_nodes_rated'] = array(
+    'description' => 'Stores information about nodes rated, thus managing the state of the context rating system',
+    'fields' => array(
+      'context_id' => array(
+        'description' => 'The ID of the context the node is rated in',
+        'type' => 'int',
+        'unsigned' => TRUE,
+        'not null' => TRUE,
+        'default' => 0,
+      ),
+      'nid' => array(
+        'description' => 'The ID of the node being rated',
+        'type' => 'int',
+        'unsigned' => TRUE,
+        'not null' => TRUE,
+        'default' => 0,
+      ),
+      'rating' => array(
+        'description' => 'The rating of the node in the context specified by context_id',
+        'type' => 'int',
+        'size' => 'tiny',
+        'unsigned' => TRUE,
+        'not null' => TRUE,
+        'default' => 1,
+      ),
+    ),
+    'primary key' => array('context_id', 'nid'),
+    'foreign keys' => array(
+      'context' => array(
+        'table' => 'ting_search_context_contexts',
+        'columns' => array('context_id' => 'context_id'),
+      ),
+      'node' => array(
+        'table' => 'node',
+        'columns' => array('nid' => 'nid'),
+      ),
+    ),
+    'indexes' => array(
+      'nid' => array('nid'),
+    ),
+  );
+
+  return $schema;
+}
+
+/**
+ * Implements hook_install().
+ */
+function ting_search_context_install() {
+  _ting_search_context_install_system_contexts();
+}
+
+/**
+ * Implements hook_unintstall().
+ */
+function ting_search_context_uninstall() {
+  variable_del('ting_search_context_position');
+  variable_del('ting_search_context_fallback_images');
+}
+
+/**
+ * Install the module's system contexts in the database.
+ *
+ * System contexts needs to live in the database, so they can be used to rate
+ * nodes in the same way as user-defined contexts (they need a context_id).
+ */
+function _ting_search_context_install_system_contexts() {
+  $system_contexts = array(
+    'film' => array(
+      'name' => 'Film',
+      'context' => 'film',
+    ),
+    'musik' => array(
+      'name' => 'Musik',
+      'context' => 'musik',
+    ),
+    'lydbøger' => array(
+      'name' => 'Lydbøger',
+      'context' => 'lydbøger',
+    ),
```

Comment:
Non-english keys and context is a bad idea, and name ought to be translated from english to danish.